pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[tool.usort]
+
+categories = [
+    "future",
+    "standard_library",
+    "third_party",
+    "pytorch",
+    "first_party",
+    "local_utils",
+]
+
+[tool.usort.known]
+
+pytorch = ["torch"]
+first_party = ["torchvision"]
+local_utils = [
+    "_utils_internal",
+    "common_utils",
+    "dataset_utils",
+]
```

Comment:
These are local utilities in our `test/` folder. Since its not a package, i.e. `tests/__init__.py` is missing, `usort` does not pick up on the fact that these are local dependencies and sorts them into the third party category.

Reply:
Let's add this comment in the file to provide context for future us :)

Also to avoid conflicting it with other utils, can we change the name to `local_test_utils`? Alternatively we could just add these into the "first-party" category?

Reply:
> Alternatively we could just add these into the "first-party" category?

We could, but if you don't have a strong opinion about this, I would leave them in an extra category. IMO, first party and local should be separated, because the latter indicates that this cannot be import the "normal" way. In fact, `isort` has a section for local imports where all the relative imports go. Unfortunately, `usort` does not have that. I've asked about it in facebookexperimental/usort#30.